portal/src/graphql/adminapi/UsersList.tsx

```diff
@@ -158,11 +177,56 @@ const PlainUsersList: React.FC<PlainUsersListProps> = function PlainUsersList(
     );
   }, []);
 
+  const onDisableUserClicked = useCallback(
+    (
+      event: React.MouseEvent<unknown>,
+      userID: string,
+      username: string | null
+    ) => {
+      // prevent triggering the link to user detail page
+      event.preventDefault();
+      event.stopPropagation();
+      setDisableUserDialogData({ userID, username });
+    },
+    []
+  );
+
+  const onRenderUserItemColumn = useCallback(
+    (item: UserListItem, _index?: number, column?: IColumn) => {
+      switch (column?.key) {
+        case "action":
+          return (
+            <ActionButton
```

Comment:
The button text is black. Can we add a theme to make it more aligned with the design?

Reply:
Added a theme to update the button text colour, but the text colour is the same as hover text colour